components/patient-data/ui/src/main/resources/PhenoTips/PatientConsentConfiguration.xml

```diff
@@ -685,7 +707,9 @@ table.extradata-list.withLabel td.collapsed * {
   height: 1px;
   overflow: hidden;
 }
-
+table input[type=text] {
```

Comment:
This is too generic, and may affect other tables. All the other rules apply to `table.extradata-list.withLabel`, so the same should be used here.